### PR TITLE
Add the options audioSource and videoSource for the quality test

### PR DESCRIPTION
**Pull request by miksansegundo**
The `NetworkTest` can receive the options `audioSource` and `videoSource` to be tested. Both, the `testConnectivity` and `testQuality` use those options to create the publisher stream. 

Usage:
```
new NetworkTest(OT, {
      apiKey,
      sessionId,
      token
}, {
      audioSource, // Microphone deviceID
      videoSource // Webcam deviceID
})

```

### CODE REVIEW

**Comment by dufrenb** [comment]
What is the status of this pull request? Can someone review it ? It feels like theses changes may solve this issue: https://github.com/opentok/opentok-network-test-js/issues/133

**Comment by cadesalaberry** [comment]
@jeffswartz do you know what is missing for this PR to be merged?

File: src/NetworkTest/testQuality/index.ts [hunk 1]
```diff
@@ -130,6 +130,13 @@ function publishAndSubscribe(OT: OT.Client) {
             insertMode: 'append',
             showControls: false,
           };
+          if (options && options.audioSource) {
+            publisherOptions.audioSource = options.audioSource
+          }
+          if (options && options.videoSource) {
+            publisherOptions.videoSource = options.videoSource
+          }
+          console.log('AQUI test', publisherOptions)
```
Comment (enricop89) [inline src/NetworkTest/testQuality/index.ts:139]: can you remove this console log, please?

File: src/NetworkTest/testConnectivity/index.ts [hunk 2]
```diff
@@ -140,6 +146,7 @@ function checkCreateLocalPublisher(
           if (!error) {
             resolve({ publisher });
           } else {
+            console.error('OT.initPublisher', publisherOptions, error)
```
Comment (enricop89) [inline src/NetworkTest/testConnectivity/index.ts:149]: can you remove this console log, please?

File: src/NetworkTest/testQuality/index.ts (hunk 1 above)
Comment (jeffswartz) [reply src/NetworkTest/testQuality/index.ts:139]: I fixed it, @enricop89.

**Comment by jeffswartz** [comment]
Thanks for the PR, @miksansegundo. Very sorry for the delay in getting this merged in. (I'm going to add some minor release notes and merge this into the master branch today.)

**Pull request merged**